nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,118 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
```

Comment:
I like this explanation, but it is a little bit contradictory to what we recommend, i.e. to treat SQL Server as a logical broker. Maybe we could state something like this:

SQL Server provides a central place to store queues and messages but the queue implementation is executed entirely within the endpoint process. As such, SQL Server transport is best thought of as a brokered transport like RabbitMQ rather than store-and-forward transport such as MSMQ.